src/MembershipsAdapter.js

```diff
@@ -11,6 +11,7 @@ import WebexAdapter from './WebexAdapter';
  * @property {boolean}  muted       If the member's audio is currently muted
  * @property {boolean}  sharing     If the member is currently sharing their screen
  * @property {boolean}  inMeeting   If the member is currently in meeting
+ * @property {boolean}  hosting     If the member is the host of a meeting
```

Comment:
Being a host is a role in a meeting, not an action so we should name it accordingly.
```suggestion
 * @property {boolean}  host     If the member is the host of a meeting
```

Reply:
Done.